web/models.py

```
@@ -2805,3 +2805,135 @@
+
+
+# Add to web/models.py
+class MassClassStream(models.Model):
+    """Model for mass virtual classroom streaming sessions"""
+
+    STATUS_CHOICES = [
+        ("initializing", "Initializing"),
+        ("active", "Active"),
+        ("ended", "Ended"),
+        ("error", "Error"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)